src/main/java/com/google/sps/servlets/ColumnsFromTablesServlet.java

```diff
@@ -35,7 +35,6 @@
 @WebServlet("/columns-from-tables")
 public class ColumnsFromTablesServlet extends HttpServlet {
     Constants constant = new Constants();
-    
     DatabaseClient dbClient;
```

Comment:
Since this is not created in the constructor, let's have this as a regular variable in the function and pass it to the functions that need it.

Reply:
done